### DISCUSSION TITLE
How to setup Colima in Windows?

### DISCUSSION THREAD

**Discussion by Tattwa** [abiosoft/colima#491] [Q&A] [answered]
I am looking for instruction or documentation of Colima installation guide for Windows OS.

**Comment by SuperFola** [comment, marked as answer]
Colima stands for containers on lima, and lima for Linux on Mac. This software isn't meant to be use on Windows.

**Reply by vilinski** [reply]
so you say this is a lie?
https://github.com/abiosoft/colima?tab=readme-ov-file#what-is-with-the-name

**Reply by abiosoft** [reply]
@vilinski it is not a lie.
The naming of Lima evolved and that led to updating the description of Colima as well. If you checked the git history of the readme files for both projects, you would see the previous descriptions.

Lima now has Windows support via WSL2 and Colima would eventually support Windows due to that.
